pkg/apis/serving/v1beta1/revision_defaults_test.go

```diff
@@ -243,6 +243,40 @@ func TestRevisionDefaulting(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "defaluting non aggressive probe",
```

Comment:
```suggestion
		name: "defaulting non aggressive probe",
```

(Don't use the commit button, please apply the change manually)